src/pages/Debug/TransactionViolation/DebugTransactionViolationCreatePage.tsx

```diff
@@ -66,6 +66,27 @@ function DebugTransactionViolationCreatePage({
     const [draftTransactionViolation, setDraftTransactionViolation] = useState<string>(getInitialTransactionViolation());
     const [error, setError] = useState<string>();
 
+    const onEditJSON = useCallback(
```

Comment:
Avoid naming with `on` prefixes for event handles, https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#event-handlers

Reply:
Thanks for the tip @fabioh8010  😄